File: app/problems/two_sum/runnable-code/execute.js
let input = "";

##USER_CODE_HERE##

process.stdin.on("data", (chunk) => {
  if (chunk !== null) {
    input += chunk;
  }
});

process.stdin.on("end", () => {
  const inputArr = input.split("\n");
  const size = parseInt(inputArr[0]);
  const k = parseInt(inputArr[1]);
  const arr = [];
  for (let i = 0; i < size; i++) {
    const num = parseInt(inputArr[i + 2]);
    arr.push(num);
  }
  const a = two_sum(arr, k)[0];
  const b = two_sum(arr, k)[1];
  const output = `${a} ${b}`;
  console.log(output);
});

File: app/problems/two_sum/runnable-code/execute.cpp
#include <iostream>
#include <cmath>
#include <string>
#include <algorithm>
#include <vector>

using namespace std;

##USER_CODE_HERE##

int main() {
    int size, k;
    cin >> size;
    cin >> k;
    vector<int> arr(size);
    for (int i = 0; i < size; i++) {
        cin >> arr[i];
    }
    vector<int> ans = two_sum(arr, k);
    for(auto& i : ans) {
        cout << i << " ";
    }
    return 0;
}